test/sources/gdal.jl

```diff
@@ -548,9 +548,14 @@ end
         gdalstack_lazy = RasterStack((a=gdalpath, b=gdalpath); lazy=true)
         @test Rasters.isdisk(gdalstack_lazy.a)
         @test Rasters.isdisk(gdalstack_lazy.b)
+        gdalstack_read = read(gdalstack_lazy)
+        read!(gdalstack_lazy, gdalstack_read)
```

Comment:
It always gets me that `read` methods have flipped arguments because of IO arg order rules...